server/main.lua

```diff
@@ -18,41 +18,42 @@ end
 local function giveStarterItems()
     local Player = QBCore.Functions.GetPlayer(source)
     if not Player then return end
+    local citizenid, firstname, lastname, gender, birthdate, nationality = Player.PlayerData.citizenid, Player.PlayerData.charinfo.firstname, Player.PlayerData.charinfo.lastname, genderString(Player.PlayerData.charinfo.gender), Player.PlayerData.charinfo.birthdate, Player.PlayerData.charinfo.nationality
     for _, v in pairs(QBCore.Shared.StarterItems) do
         if v.item == 'id_card' then
             local metadata = {
-                type = string.format('%s %s', Player.PlayerData.charinfo.firstname, Player.PlayerData.charinfo.lastname),
-                description = string.format('CID: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', Player.PlayerData.citizenid, Player.PlayerData.charinfo.birthdate, genderString(Player.PlayerData.charinfo.gender), Player.PlayerData.charinfo.nationality),
-                citizenid = Player.PlayerData.citizenid,
-                firstname = Player.PlayerData.charinfo.firstname,
-                lastname = Player.PlayerData.charinfo.lastname,
-                gender = genderString(Player.PlayerData.charinfo.gender),
-                birthdate = Player.PlayerData.charinfo.birthdate,
-                nationality = Player.PlayerData.charinfo.nationality,
+                type = string.format('%s %s', firstname, lastname),
+                description = string.format('CID: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', citizenid, birthdate, gender, nationality),
+                citizenid = citizenid,
+                firstname = firstname,
+                lastname = lastname,
+                gender = gender,
+                birthdate = birthdate,
+                nationality = nationality,
```

Comment:
This is an improvement, but it could still be improved even more by declaring the metadata variable outside the conditionals and then modifying the metadata within the conditionals. That way the above code need not be duplicated in each conditional.

Reply:
not understanding or give me a short direction then yes I am down to make it more more more nicer

Reply:
Move the declaration of the metadata variable outside the conditionals.